### ISSUE TITLE
User password should be hashed

### ISSUE THREAD

**Issue by KrzysztofMadejski** [TransparenCEE/factchecker-wordpress-api#4]
See http://www.factual.ro/api/administrative/admin_users_add_modify.php?edit_id=2

This is super bad and dangerous practice. Having plain passwords in the database shows them to the admin - this shouldn't be possible (many ppl reuse passwords on different websites). Also when database is leaked / hacked that shows passwords to crackers.

Apart from the hash the secure salt (taken from configuration should be added). Otherwise leaked passwords can be easily checked with big database of prehashed passwords (lookup tables)

Here is a longer read: https://crackstation.net/hashing-security.htm

In short: please implement http://www.openwall.com/phpass/ :)

**Comment by catalintenita** [comment]
Ok, issue solved.

**Comment by KrzysztofMadejski** [comment]
Thank you
